src/cnd/axios_rfc7807_middleware.spec.ts

```diff
@@ -0,0 +1,49 @@
+import { Problem } from "./axios_rfc7807_middleware";
+
+describe("problem message", () => {
+  it("should format all fields to a descriptive message", () => {
+    const problem = new Problem({
+      title: "Swap not supported.",
+      status: 400,
+      detail: "This combination of protocols is not supported.",
+      type: "https://comit.network/docs/errors/swap-not-supported"
```

Comment:
This page does not exist, are we planning to add it?
Should this be a link to the sdk documentation?

Reply:
This is just test data, so I wouldn't care too much.

I can replace it with something completely non-nonsensical (like `example.com`) if you want :)